go/vt/sqlparser/sql.y

```diff
@@ -112,6 +112,7 @@ func forceEOF(yylex interface{}) {
   vindexParam   VindexParam
   vindexParams  []VindexParam
   showFilter    *ShowFilter
+  OptLike       *OptLike
```

Comment:
For uniformity, let's use `optLike *OptLike`. I know a few people have digressed, but we can fix those later.